pip/qsharp/_native.pyi

```diff
@@ -4,6 +4,14 @@
 from enum import Enum
 from typing import Any, Callable, ClassVar, Optional, Dict, List
 
+class TargetCapabilities(Enum):
```

Comment:
Are these used anywhere yet? It seems like they are aren't.

Reply:
Good catch, this slipped through the cracks.